README.md

```diff
@@ -40,8 +40,15 @@ _\* When using Octal Flash and/or Octal PSRAM_
 ## Examples
 
 The examples could be built and flashed conveniently with [`cargo-espflash`](https://github.com/esp-rs/espflash/). To run `ledc_simple` on an ESP32-C3:
+
+with cargo-esptool v1.7:
+```
+$ ESP_IDF_VERSION=release/v4.4 cargo espflash --target xtensa-esp32-espidf --example ledc_simple --monitor /dev/ttyUSB0
+```
+
+with cargo-esptool v2.0:
 ```
-$ cargo espflash --release --target riscv32imc-esp-espidf --example ledc_simple --monitor /dev/ttyUSB0
+$ ESP_IDF_VERSION=release/v4.4 cargo espflash flash --target xtensa-esp32-espidf --example ledc_simple --monitor
```

Comment:
The target for `ESP32-C3` (which is mentioned above) *must* be `riscv32imc-esp-espidf` so if you can restore that.

Also you might want to replace "To run `ledc_simple` on an ESP32-C3" with "To run e.g. `ledc_simple` e.g. on an ESP32-C3" to make it clear, that the same command line (modulo the target name and the example name) can be used for other MCUs and for other examples.

And you can add

Reply:
done